src/ComputeServer.java

```diff
@@ -0,0 +1,43 @@
+import io.grpc.Grpc;
+import io.grpc.InsecureServerCredentials;
+import io.grpc.Server;
+import io.grpc.protobuf.services.ProtoReflectionService;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+public class ComputeServer {
+  private Server server;
+
+  void start() throws IOException {
+    int port = 8010;

```

Comment:
maybe change this to final int port = 8010

Reply:
Sure, done